### PR TITLE
SW-2103 Add Planting Sites empty state

### PR DESCRIPTION
**Pull request by constanzauanini**
Screenshot 2022-11-09 at 12 27 24

TODO: Replace with correct icon

### CODE REVIEW

File: src/components/PlantingSites/index.tsx [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import { Box } from '@mui/material';
-import strings from 'src/strings';
+import EmptyStatePage from '../emptyStatePages/EmptyStatePage';
 
 export default function PlantingSitesList(): JSX.Element {
-  return <Box>{strings.PLANTING_SITES}</Box>;
+  return (
+    <Box>
```
Comment (karthikbtf) [inline src/components/PlantingSites/index.tsx:6]: i think we may need to add the same components like in seedbanks / nurseries etc.. there's some extra padding in the planting sites view.
fwiw, i think the extra padding in planting sites view looks cleaner - it helps fix https://terraformation.atlassian.net/browse/SW-2187 but not sure what the spec is.

**Review by karthikbtf — APPROVED** [review]
@constanzauanini two issues I noticed, both unrelated to this PR but just FYI.
1) planting sites menu does not show up for contributors
2) empty state content looks different, padding/positioning for various pages in various scenarios
Not sure how we should proceed regarding 2)
Btw, the search api PR for planting sites in BE was approved - should go in soon.

File: src/components/PlantingSites/index.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/PlantingSites/index.tsx:6]: Mmm but seedbanks and nurseries have the emptyComponent called directly from the app.tsx. I could add the same components as inventory that is the only one calling the empty state inside the component

**Review by karthikbtf — APPROVED** [review]
Thanks @constanzauanini

**Pull request merged**